spec/live/cbp_support.rb

```diff
@@ -0,0 +1,37 @@
+require 'core/spec_helper'
+require 'core/cbp_helper'
+describe 'Endpoints that support CBP' do
+  describe ZendeskAPI::Group do
+    describe '/groups' do
+      expect_cbp_response_for(client.groups)
+      expect_cbp_response_for(client.groups.assignable)
+    end
+  end
+
+  describe ZendeskAPI::GroupMembership do
+    describe '/groups/{id}/memberships' do
+      before do
+        VCR.use_cassette("cbp_group_memberships_all_groups") do
+          @groups = client.groups.fetch
```

Comment:
Could we use `let` instead of instance vars?

https://www.betterspecs.org/#let

Reply:
Done it ✅